Formula/restic.rb

```diff
@@ -4,6 +4,7 @@ class Restic < Formula
   url "https://github.com/restic/restic/archive/v0.13.1.tar.gz"
   sha256 "8430f80dc17b98fd78aca6f7d635bf12a486687677e15989a891ff4f6d8490a9"
   license "BSD-2-Clause"
+  revision 1
```

Comment:
A revision bump just for fish completions seems a bit much.

Reply:
I would've changed it, but it's already merged now, but I'll keep that in mind in the future.